speech/audioutils.py

```diff
@@ -15,16 +15,10 @@ def get_audio_commands(text, outfile, lang, cache_path, speed):
     overflow_len = 30000
     cmds = []
     names = []
-    # remove parenthesis to avoid bugs with pico2wave command
-    text = text.replace('"', '')
-    text = text.replace("'", '')
```

Comment:
can you remove this 3 lines and send it to another PR. string pass to picowave seems not exactly the same

Reply:
It seems unnecessary to me with subprocess

Reply:
double quote replacing is probably useless because treated previously but simple quote not.
I don't undersand relationship with subprocess.